web/packages/build/package.json

```diff
@@ -10,30 +10,30 @@
     "directory": "packages/build"
   },
   "dependencies": {
-    "@babel/core": "^7.28.0",
-    "@babel/preset-env": "^7.28.0",
+    "@babel/core": "^7.28.3",
+    "@babel/preset-env": "^7.28.3",
     "@babel/preset-react": "^7.27.1",
     "@babel/preset-typescript": "^7.27.1",
-    "@eslint/js": "^9.32.0",
-    "@swc/core": "^1.13.3",
-    "@swc/plugin-styled-components": "^9.0.2",
+    "@eslint/js": "^9.34.0",
+    "@swc/core": "^1.13.5",
+    "@swc/plugin-styled-components": "^9.0.3",
     "@types/jsdom": "^21.1.7",
     "@vitejs/plugin-react-swc": "^3.11.0",
     "babel-plugin-styled-components": "^2.1.4",
-    "eslint": "^9.32.0",
+    "eslint": "^9.34.0",
     "eslint-plugin-jest": "^29.0.1",
     "eslint-plugin-jest-dom": "^5.5.0",
     "eslint-plugin-react": "^7.37.5",
     "eslint-plugin-react-hooks": "6.0.0-rc.1",
-    "eslint-plugin-storybook": "^9.1.1",
+    "eslint-plugin-storybook": "^9.1.3",
     "eslint-plugin-testing-library": "7.4.0",
```

Comment:
We'll have to evaluate if we want to have `no-node-access` turned on in the first place. Maybe it has config options that make it more palatable, but I found that with this rule in place there's no way to test if a certain section of HTML includes a specific text (see https://github.com/gravitational/teleport/pull/57864#discussion_r2288453750).

Perhaps this style of testing itself is wrong in some way, but I see no problem with it as long as it doesn't use test IDs and depends on semantic HTML elements instead.

Reply:
I'm fine with disabling the rule, I see the problems with it.
I guess we can do this separately as we will have to remove all unused eslint-disable directives.